contentTypesRouter.js

```diff
@@ -0,0 +1,190 @@
+const path = require("path");
+const fs = require("fs").promises;
+const express = require("express");
+const { orderBy, slice } = require("lodash");
+const gitUtils = require("./git-utils");
+const utils = require("./utils/common");
+const media = require("./utils/media");
+
+const { getContentType } = utils;
+
+module.exports = (root, gitRepositoryRoot, publicBaseUrl) => {
+  const { fileFieldsAppendSrc } = media(root, publicBaseUrl);
+
+  async function getMany(req, res) {
```

Comment:
nit-picky: 

I'd prefer to avoid large function wrappers and pass required parameters via bind. It's more transparent as required params are passed explicitly.

```js

async function getMany(root, gitRepositoryRoot, publicBaseUrl, req, res) {...}
...

module.exports = (root, gitRepositoryRoot, publicBaseUrl) => {
   ...
  router.get("/:contentType", getMany.bind(null, root, gitRepositoryRoot, publicBaseUrl));
}


```